test_regression/test_doc_samples/test_simple_triangulation_sample.py

```diff
@@ -23,16 +21,23 @@ def test_simple_traingulation_sample(tmp_path):
     # Prepare for MeshLib PointCloud
     verts = np.stack((x.flatten(), y.flatten(), z.flatten()), axis=-1).reshape(-1, 3)
     # Create MeshLib PointCloud from np ndarray
-    pc = mn.pointCloudFromPoints(verts)
+    pc = mrmeshnumpy.pointCloudFromPoints(verts)
     # Remove duplicate points
-    pc.validPoints = mm.pointUniformSampling(pc, 1e-3)
+    if is_new_binding:
```

Comment:
This folder "test_doc_samples" reflects the code samples, that placed in out docs. Non-trivial changes of this tests makes this tests meaningless

Reply:
Agreed to fix this later, when fully switching to new bindings.